## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 471098`
Commit: `b49491a0308c4b7e4bbec0afc0cb831f87d1043b`

---
### `aarch64-darwin`
<details>
  <summary>:fast_forward: 2 packages marked as broken and skipped:</summary>
  <ul>
    <li>python314Packages.fastmcp</li>
    <li>python314Packages.fastmcp.dist</li>
  </ul>
</details>
<details>
  <summary>:x: 30 packages failed to build:</summary>
  <ul>
    <li>lasuite-docs</li>
    <li>lasuite-docs.dist</li>
    <li>python313Packages.ax-platform</li>
    <li>python313Packages.ax-platform.dist</li>
    <li>python314Packages.banks</li>
    <li>python314Packages.banks.dist</li>
    <li>python314Packages.compressai</li>
    <li>python314Packages.compressai.dist</li>
    <li>python314Packages.fnllm</li>
    <li>python314Packages.fnllm.dist</li>
    <li>python314Packages.jupysql</li>
    <li>python314Packages.jupysql-plugin</li>
    <li>python314Packages.jupysql-plugin.dist</li>
    <li>python314Packages.jupysql.dist</li>
    <li>python314Packages.livelossplot</li>
    <li>python314Packages.livelossplot.dist</li>
    <li>python314Packages.ploomber-core</li>
    <li>python314Packages.ploomber-core.dist</li>
    <li>python314Packages.ploomber-extension</li>
    <li>python314Packages.ploomber-extension.dist</li>
    <li>python314Packages.reflex</li>
    <li>python314Packages.reflex-chakra</li>
    <li>python314Packages.reflex-chakra.dist</li>
    <li>python314Packages.reflex.dist</li>
    <li>python314Packages.skrl</li>
    <li>python314Packages.skrl.dist</li>
    <li>python314Packages.spacy-loggers</li>
    <li>python314Packages.spacy-loggers.dist</li>
    <li>python314Packages.wandb</li>
    <li>python314Packages.wandb.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 693 packages built:</summary>
  <ul>
    <li>aider-chat</li>
    <li>aider-chat-full</li>
    <li>aider-chat-full.dist</li>
    <li>aider-chat-with-bedrock</li>
    <li>aider-chat-with-bedrock.dist</li>
    <li>aider-chat-with-browser</li>
    <li>aider-chat-with-browser.dist</li>
    <li>aider-chat-with-help</li>
    <li>aider-chat-with-help.dist</li>
    <li>aider-chat-with-playwright</li>
    <li>aider-chat-with-playwright.dist</li>
    <li>aider-chat.dist</li>
    <li>apkleaks</li>
    <li>apkleaks.dist</li>
    <li>azure-cli-extensions.rdbms-connect</li>
    <li>azure-cli-extensions.rdbms-connect.dist</li>
    <li>chatblade</li>
    <li>chatblade.dist</li>
    <li>checkov</li>
    <li>checkov.dist</li>
    <li>cve-bin-tool</li>
    <li>cve-bin-tool.dist</li>
    <li>das</li>
    <li>das.dist</li>
    <li>deeptools</li>
    <li>deeptools.dist</li>
    <li>flattenReferencesGraph</li>
    <li>flattenReferencesGraph.dist</li>
    <li>gpt-cli</li>
    <li>gpt-cli.dist</li>
    <li>jadx</li>
    <li>lacus</li>
    <li>lacus.dist</li>
    <li>litellm</li>
    <li>litellm.dist</li>
    <li>llm</li>
    <li>marimo (python313Packages.marimo)</li>
    <li>marimo.dist (python313Packages.marimo.dist)</li>
    <li>mcp-nixos</li>
    <li>mcp-nixos.dist</li>
    <li>mealie</li>
    <li>mealie.dist</li>
    <li>mlflow-server</li>
    <li>mlflow-server.dist</li>
    <li>multiqc</li>
    <li>multiqc.dist</li>
    <li>mycli</li>
    <li>mycli.dist</li>
    <li>openai (python313Packages.openai)</li>
    <li>openai.dist (python313Packages.openai.dist)</li>
    <li>oterm</li>
    <li>oterm.dist</li>
    <li>pentestgpt</li>
    <li>pentestgpt.dist</li>
    <li>prowler</li>
    <li>prowler.dist</li>
    <li>python313Packages.altair</li>
    <li>python313Packages.altair.dist</li>
    <li>python313Packages.backtesting</li>
    <li>python313Packages.backtesting.dist</li>
    <li>python313Packages.banks</li>
    <li>python313Packages.banks.dist</li>
    <li>python313Packages.bokeh</li>
    <li>python313Packages.bokeh.dist</li>
    <li>python313Packages.bumps</li>
    <li>python313Packages.bumps.dist</li>
    <li>python313Packages.chart-studio</li>
    <li>python313Packages.chart-studio.dist</li>
    <li>python313Packages.chatlas</li>
    <li>python313Packages.chatlas.dist</li>
    <li>python313Packages.chromadb</li>
    <li>python313Packages.chromadb.dist</li>
    <li>python313Packages.cltk</li>
    <li>python313Packages.cltk.dist</li>
    <li>python313Packages.coiled</li>
    <li>python313Packages.coiled.dist</li>
    <li>python313Packages.compressai</li>
    <li>python313Packages.compressai.dist</li>
    <li>python313Packages.dash</li>
    <li>python313Packages.dash-bootstrap-components</li>
    <li>python313Packages.dash-bootstrap-components.dist</li>
    <li>python313Packages.dash-bootstrap-templates</li>
    <li>python313Packages.dash-bootstrap-templates.dist</li>
    <li>python313Packages.dash.dist</li>
    <li>python313Packages.databricks-sdk</li>
    <li>python313Packages.databricks-sdk.dist</li>
    <li>python313Packages.energyflow</li>
    <li>python313Packages.energyflow.dist</li>
    <li>python313Packages.experiment-utilities</li>
    <li>python313Packages.experiment-utilities.dist</li>
    <li>python313Packages.explorerscript</li>
    <li>python313Packages.explorerscript.dist</li>
    <li>python313Packages.extra-streamlit-components</li>
    <li>python313Packages.extra-streamlit-components.dist</li>
    <li>python313Packages.fastai</li>
    <li>python313Packages.fastai.dist</li>
    <li>python313Packages.fastmcp</li>
    <li>python313Packages.fastmcp.dist</li>
    <li>python313Packages.fireworks-ai</li>
    <li>python313Packages.fireworks-ai.dist</li>
    <li>python313Packages.fnllm</li>
    <li>python313Packages.fnllm.dist</li>
    <li>python313Packages.formulaic</li>
    <li>python313Packages.formulaic.dist</li>
    <li>python313Packages.gradio</li>
    <li>python313Packages.gradio.dist</li>
    <li>python313Packages.great-tables</li>
    <li>python313Packages.great-tables.dist</li>
    <li>python313Packages.holistic-trace-analysis</li>
    <li>python313Packages.holistic-trace-analysis.dist</li>
    <li>python313Packages.holoviews</li>
    <li>python313Packages.holoviews.dist</li>
    <li>python313Packages.hvplot</li>
    <li>python313Packages.hvplot.dist</li>
    <li>python313Packages.igraph</li>
    <li>python313Packages.igraph.dist</li>
    <li>python313Packages.instructor</li>
    <li>python313Packages.instructor.dist</li>
    <li>python313Packages.intake</li>
    <li>python313Packages.intake.dist</li>
    <li>python313Packages.iplotx</li>
    <li>python313Packages.iplotx.dist</li>
    <li>python313Packages.itables</li>
    <li>python313Packages.itables.dist</li>
    <li>python313Packages.jupysql</li>
    <li>python313Packages.jupysql-plugin</li>
    <li>python313Packages.jupysql-plugin.dist</li>
    <li>python313Packages.jupysql.dist</li>
    <li>python313Packages.k-diffusion</li>
    <li>python313Packages.k-diffusion.dist</li>
    <li>python313Packages.kmapper</li>
    <li>python313Packages.kmapper.dist</li>
    <li>python313Packages.lacuscore</li>
    <li>python313Packages.lacuscore.dist</li>
    <li>python313Packages.langchain-azure-dynamic-sessions</li>
    <li>python313Packages.langchain-azure-dynamic-sessions.dist</li>
    <li>python313Packages.langchain-chroma</li>
    <li>python313Packages.langchain-chroma.dist</li>
    <li>python313Packages.langchain-classic</li>
    <li>python313Packages.langchain-classic.dist</li>
    <li>python313Packages.langchain-community</li>
    <li>python313Packages.langchain-community.dist</li>
    <li>python313Packages.langchain-deepseek</li>
    <li>python313Packages.langchain-deepseek.dist</li>
    <li>python313Packages.langchain-experimental</li>
    <li>python313Packages.langchain-experimental.dist</li>
    <li>python313Packages.langchain-fireworks</li>
    <li>python313Packages.langchain-fireworks.dist</li>
    <li>python313Packages.langchain-mongodb</li>
    <li>python313Packages.langchain-mongodb.dist</li>
    <li>python313Packages.langchain-openai</li>
    <li>python313Packages.langchain-openai.dist</li>
    <li>python313Packages.langchain-perplexity</li>
    <li>python313Packages.langchain-perplexity.dist</li>
    <li>python313Packages.langchain-xai</li>
    <li>python313Packages.langchain-xai.dist</li>
    <li>python313Packages.langgraph-checkpoint-mongodb</li>
    <li>python313Packages.langgraph-checkpoint-mongodb.dist</li>
    <li>python313Packages.langgraph-checkpoint-sqlite</li>
    <li>python313Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python313Packages.langgraph-store-mongodb</li>
    <li>python313Packages.langgraph-store-mongodb.dist</li>
    <li>python313Packages.layoutparser</li>
    <li>python313Packages.layoutparser.dist</li>
    <li>python313Packages.leidenalg</li>
    <li>python313Packages.leidenalg.dist</li>
    <li>python313Packages.lifelines</li>
    <li>python313Packages.lifelines.dist</li>
    <li>python313Packages.linearmodels</li>
    <li>python313Packages.linearmodels.dist</li>
    <li>python313Packages.litellm</li>
    <li>python313Packages.litellm.dist</li>
    <li>python313Packages.livelossplot</li>
    <li>python313Packages.livelossplot.dist</li>
    <li>python313Packages.llama-cloud-services</li>
    <li>python313Packages.llama-cloud-services.dist</li>
    <li>python313Packages.llama-index</li>
    <li>python313Packages.llama-index-cli</li>
    <li>python313Packages.llama-index-cli.dist</li>
    <li>python313Packages.llama-index-core</li>
    <li>python313Packages.llama-index-core.dist</li>
    <li>python313Packages.llama-index-embeddings-gemini</li>
    <li>python313Packages.llama-index-embeddings-gemini.dist</li>
    <li>python313Packages.llama-index-embeddings-google</li>
    <li>python313Packages.llama-index-embeddings-google.dist</li>
    <li>python313Packages.llama-index-embeddings-huggingface</li>
    <li>python313Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python313Packages.llama-index-embeddings-ollama</li>
    <li>python313Packages.llama-index-embeddings-ollama.dist</li>
    <li>python313Packages.llama-index-embeddings-openai</li>
    <li>python313Packages.llama-index-embeddings-openai.dist</li>
    <li>python313Packages.llama-index-graph-stores-neo4j</li>
    <li>python313Packages.llama-index-graph-stores-neo4j.dist</li>
    <li>python313Packages.llama-index-graph-stores-neptune</li>
    <li>python313Packages.llama-index-graph-stores-neptune.dist</li>
    <li>python313Packages.llama-index-indices-managed-llama-cloud</li>
    <li>python313Packages.llama-index-indices-managed-llama-cloud.dist</li>
    <li>python313Packages.llama-index-legacy</li>
    <li>python313Packages.llama-index-legacy.dist</li>
    <li>python313Packages.llama-index-llms-ollama</li>
    <li>python313Packages.llama-index-llms-ollama.dist</li>
    <li>python313Packages.llama-index-llms-openai</li>
    <li>python313Packages.llama-index-llms-openai-like</li>
    <li>python313Packages.llama-index-llms-openai-like.dist</li>
    <li>python313Packages.llama-index-llms-openai.dist</li>
    <li>python313Packages.llama-index-multi-modal-llms-openai</li>
    <li>python313Packages.llama-index-multi-modal-llms-openai.dist</li>
    <li>python313Packages.llama-index-node-parser-docling</li>
    <li>python313Packages.llama-index-node-parser-docling.dist</li>
    <li>python313Packages.llama-index-readers-database</li>
    <li>python313Packages.llama-index-readers-database.dist</li>
    <li>python313Packages.llama-index-readers-file</li>
    <li>python313Packages.llama-index-readers-file.dist</li>
    <li>python313Packages.llama-index-readers-json</li>
    <li>python313Packages.llama-index-readers-json.dist</li>
    <li>python313Packages.llama-index-readers-llama-parse</li>
    <li>python313Packages.llama-index-readers-llama-parse.dist</li>
    <li>python313Packages.llama-index-readers-s3</li>
    <li>python313Packages.llama-index-readers-s3.dist</li>
    <li>python313Packages.llama-index-readers-twitter</li>
    <li>python313Packages.llama-index-readers-twitter.dist</li>
    <li>python313Packages.llama-index-readers-txtai</li>
    <li>python313Packages.llama-index-readers-txtai.dist</li>
    <li>python313Packages.llama-index-readers-weather</li>
    <li>python313Packages.llama-index-readers-weather.dist</li>
    <li>python313Packages.llama-index-vector-stores-chroma</li>
    <li>python313Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python313Packages.llama-index-vector-stores-google</li>
    <li>python313Packages.llama-index-vector-stores-google.dist</li>
    <li>python313Packages.llama-index-vector-stores-milvus</li>
    <li>python313Packages.llama-index-vector-stores-milvus.dist</li>
    <li>python313Packages.llama-index-vector-stores-qdrant</li>
    <li>python313Packages.llama-index-vector-stores-qdrant.dist</li>
    <li>python313Packages.llama-index.dist</li>
    <li>python313Packages.llama-parse</li>
    <li>python313Packages.llama-parse.dist</li>
    <li>python313Packages.llm</li>
    <li>python313Packages.llm-anthropic</li>
    <li>python313Packages.llm-anthropic.dist</li>
    <li>python313Packages.llm-cmd</li>
    <li>python313Packages.llm-cmd.dist</li>
    <li>python313Packages.llm-command-r</li>
    <li>python313Packages.llm-command-r.dist</li>
    <li>python313Packages.llm-deepseek</li>
    <li>python313Packages.llm-deepseek.dist</li>
    <li>python313Packages.llm-docs</li>
    <li>python313Packages.llm-docs.dist</li>
    <li>python313Packages.llm-echo</li>
    <li>python313Packages.llm-echo.dist</li>
    <li>python313Packages.llm-fragments-github</li>
    <li>python313Packages.llm-fragments-github.dist</li>
    <li>python313Packages.llm-fragments-pypi</li>
    <li>python313Packages.llm-fragments-pypi.dist</li>
    <li>python313Packages.llm-fragments-reader</li>
    <li>python313Packages.llm-fragments-reader.dist</li>
    <li>python313Packages.llm-fragments-symbex</li>
    <li>python313Packages.llm-fragments-symbex.dist</li>
    <li>python313Packages.llm-gemini</li>
    <li>python313Packages.llm-gemini.dist</li>
    <li>python313Packages.llm-gguf</li>
    <li>python313Packages.llm-gguf.dist</li>
    <li>python313Packages.llm-git</li>
    <li>python313Packages.llm-git.dist</li>
    <li>python313Packages.llm-github-copilot</li>
    <li>python313Packages.llm-github-copilot.dist</li>
    <li>python313Packages.llm-grok</li>
    <li>python313Packages.llm-grok.dist</li>
    <li>python313Packages.llm-groq</li>
    <li>python313Packages.llm-groq.dist</li>
    <li>python313Packages.llm-hacker-news</li>
    <li>python313Packages.llm-hacker-news.dist</li>
    <li>python313Packages.llm-jq</li>
    <li>python313Packages.llm-jq.dist</li>
    <li>python313Packages.llm-llama-server</li>
    <li>python313Packages.llm-llama-server.dist</li>
    <li>python313Packages.llm-mistral</li>
    <li>python313Packages.llm-mistral.dist</li>
    <li>python313Packages.llm-ollama</li>
    <li>python313Packages.llm-ollama.dist</li>
    <li>python313Packages.llm-openai-plugin</li>
    <li>python313Packages.llm-openai-plugin.dist</li>
    <li>python313Packages.llm-openrouter</li>
    <li>python313Packages.llm-openrouter.dist</li>
    <li>python313Packages.llm-pdf-to-images</li>
    <li>python313Packages.llm-pdf-to-images.dist</li>
    <li>python313Packages.llm-perplexity</li>
    <li>python313Packages.llm-perplexity.dist</li>
    <li>python313Packages.llm-sentence-transformers</li>
    <li>python313Packages.llm-sentence-transformers.dist</li>
    <li>python313Packages.llm-templates-fabric</li>
    <li>python313Packages.llm-templates-fabric.dist</li>
    <li>python313Packages.llm-templates-github</li>
    <li>python313Packages.llm-templates-github.dist</li>
    <li>python313Packages.llm-tools-datasette</li>
    <li>python313Packages.llm-tools-datasette.dist</li>
    <li>python313Packages.llm-tools-quickjs</li>
    <li>python313Packages.llm-tools-quickjs.dist</li>
    <li>python313Packages.llm-tools-simpleeval</li>
    <li>python313Packages.llm-tools-simpleeval.dist</li>
    <li>python313Packages.llm-tools-sqlite</li>
    <li>python313Packages.llm-tools-sqlite.dist</li>
    <li>python313Packages.llm-venice</li>
    <li>python313Packages.llm-venice.dist</li>
    <li>python313Packages.llm-video-frames</li>
    <li>python313Packages.llm-video-frames.dist</li>
    <li>python313Packages.llm.dist</li>
    <li>python313Packages.llmx</li>
    <li>python313Packages.llmx.dist</li>
    <li>python313Packages.markitdown</li>
    <li>python313Packages.markitdown.dist</li>
    <li>python313Packages.mistral-common</li>
    <li>python313Packages.mistral-common.dist</li>
    <li>python313Packages.mlcroissant</li>
    <li>python313Packages.mlcroissant.dist</li>
    <li>python313Packages.mlflow</li>
    <li>python313Packages.mlflow.dist</li>
    <li>python313Packages.mmcv</li>
    <li>python313Packages.mmcv.dist</li>
    <li>python313Packages.mmengine</li>
    <li>python313Packages.mmengine.dist</li>
    <li>python313Packages.msticpy</li>
    <li>python313Packages.msticpy.dist</li>
    <li>python313Packages.narwhals</li>
    <li>python313Packages.narwhals.dist</li>
    <li>python313Packages.nbsmoke</li>
    <li>python313Packages.nbsmoke.dist</li>
    <li>python313Packages.neurokit2</li>
    <li>python313Packages.neurokit2.dist</li>
    <li>python313Packages.niaarm</li>
    <li>python313Packages.niaarm.dist</li>
    <li>python313Packages.openai-agents</li>
    <li>python313Packages.openai-agents.dist</li>
    <li>python313Packages.optuna</li>
    <li>python313Packages.optuna-dashboard</li>
    <li>python313Packages.optuna-dashboard.dist</li>
    <li>python313Packages.optuna.dist</li>
    <li>python313Packages.outlines</li>
    <li>python313Packages.outlines.dist</li>
    <li>python313Packages.pandantic</li>
    <li>python313Packages.pandantic.dist</li>
    <li>python313Packages.pandas-stubs</li>
    <li>python313Packages.pandas-stubs.dist</li>
    <li>python313Packages.pandera</li>
    <li>python313Packages.pandera.dist</li>
    <li>python313Packages.panel</li>
    <li>python313Packages.panel.dist</li>
    <li>python313Packages.pdfplumber</li>
    <li>python313Packages.pdfplumber.dist</li>
    <li>python313Packages.ploomber-core</li>
    <li>python313Packages.ploomber-core.dist</li>
    <li>python313Packages.ploomber-extension</li>
    <li>python313Packages.ploomber-extension.dist</li>
    <li>python313Packages.plotly</li>
    <li>python313Packages.plotly.dist</li>
    <li>python313Packages.posthog</li>
    <li>python313Packages.posthog.dist</li>
    <li>python313Packages.pyannote-audio</li>
    <li>python313Packages.pyannote-audio.dist</li>
    <li>python313Packages.pyannote-pipeline</li>
    <li>python313Packages.pyannote-pipeline.dist</li>
    <li>python313Packages.pymoo</li>
    <li>python313Packages.pymoo.dist</li>
    <li>python313Packages.pypasser</li>
    <li>python313Packages.pypasser.dist</li>
    <li>python313Packages.pyreadstat</li>
    <li>python313Packages.pyreadstat.dist</li>
    <li>python313Packages.reflex</li>
    <li>python313Packages.reflex-chakra</li>
    <li>python313Packages.reflex-chakra.dist</li>
    <li>python313Packages.reflex.dist</li>
    <li>python313Packages.sagemaker-mlflow</li>
    <li>python313Packages.sagemaker-mlflow.dist</li>
    <li>python313Packages.sasmodels</li>
    <li>python313Packages.sasmodels.dist</li>
    <li>python313Packages.scikit-tda</li>
    <li>python313Packages.scikit-tda.dist</li>
    <li>python313Packages.shiny</li>
    <li>python313Packages.shiny.dist</li>
    <li>python313Packages.skops</li>
    <li>python313Packages.skops.dist</li>
    <li>python313Packages.skrl</li>
    <li>python313Packages.skrl.dist</li>
    <li>python313Packages.smolagents</li>
    <li>python313Packages.smolagents.dist</li>
    <li>python313Packages.spacy</li>
    <li>python313Packages.spacy-curated-transformers</li>
    <li>python313Packages.spacy-curated-transformers.dist</li>
    <li>python313Packages.spacy-loggers</li>
    <li>python313Packages.spacy-loggers.dist</li>
    <li>python313Packages.spacy-lookups-data</li>
    <li>python313Packages.spacy-lookups-data.dist</li>
    <li>python313Packages.spacy-transformers</li>
    <li>python313Packages.spacy-transformers.dist</li>
    <li>python313Packages.spacy.dist</li>
    <li>python313Packages.speechrecognition</li>
    <li>python313Packages.speechrecognition.dist</li>
    <li>python313Packages.sqlframe</li>
    <li>python313Packages.sqlframe.dist</li>
    <li>python313Packages.sqlite-vec</li>
    <li>python313Packages.sqlite-vec.dist</li>
    <li>python313Packages.st-pages</li>
    <li>python313Packages.st-pages.dist</li>
    <li>streamlit (python313Packages.streamlit)</li>
    <li>python313Packages.streamlit-avatar</li>
    <li>python313Packages.streamlit-avatar.dist</li>
    <li>python313Packages.streamlit-card</li>
    <li>python313Packages.streamlit-card.dist</li>
    <li>python313Packages.streamlit-echarts</li>
    <li>python313Packages.streamlit-echarts.dist</li>
    <li>python313Packages.streamlit-folium</li>
    <li>python313Packages.streamlit-folium.dist</li>
    <li>python313Packages.streamlit-kpi-card</li>
    <li>python313Packages.streamlit-kpi-card.dist</li>
    <li>python313Packages.streamlit-notify</li>
    <li>python313Packages.streamlit-notify.dist</li>
    <li>streamlit.dist (python313Packages.streamlit.dist)</li>
    <li>python313Packages.synergy</li>
    <li>python313Packages.synergy.dist</li>
    <li>python313Packages.tami4edgeapi</li>
    <li>python313Packages.tami4edgeapi.dist</li>
    <li>python313Packages.textacy</li>
    <li>python313Packages.textacy.dist</li>
    <li>python313Packages.textnets</li>
    <li>python313Packages.textnets.dist</li>
    <li>python313Packages.torchtune</li>
    <li>python313Packages.torchtune.dist</li>
    <li>python313Packages.unstructured-inference</li>
    <li>python313Packages.unstructured-inference.dist</li>
    <li>python313Packages.vega</li>
    <li>python313Packages.vega.dist</li>
    <li>python313Packages.wandb</li>
    <li>python313Packages.wandb.dist</li>
    <li>whisperx (python313Packages.whisperx)</li>
    <li>whisperx.dist (python313Packages.whisperx.dist)</li>
    <li>python314Packages.altair</li>
    <li>python314Packages.altair.dist</li>
    <li>python314Packages.backtesting</li>
    <li>python314Packages.backtesting.dist</li>
    <li>python314Packages.bokeh</li>
    <li>python314Packages.bokeh.dist</li>
    <li>python314Packages.bumps</li>
    <li>python314Packages.bumps.dist</li>
    <li>python314Packages.chart-studio</li>
    <li>python314Packages.chart-studio.dist</li>
    <li>python314Packages.chatlas</li>
    <li>python314Packages.chatlas.dist</li>
    <li>python314Packages.coiled</li>
    <li>python314Packages.coiled.dist</li>
    <li>python314Packages.dash</li>
    <li>python314Packages.dash-bootstrap-components</li>
    <li>python314Packages.dash-bootstrap-components.dist</li>
    <li>python314Packages.dash-bootstrap-templates</li>
    <li>python314Packages.dash-bootstrap-templates.dist</li>
    <li>python314Packages.dash.dist</li>
    <li>python314Packages.databricks-sdk</li>
    <li>python314Packages.databricks-sdk.dist</li>
    <li>python314Packages.experiment-utilities</li>
    <li>python314Packages.experiment-utilities.dist</li>
    <li>python314Packages.explorerscript</li>
    <li>python314Packages.explorerscript.dist</li>
    <li>python314Packages.extra-streamlit-components</li>
    <li>python314Packages.extra-streamlit-components.dist</li>
    <li>python314Packages.fireworks-ai</li>
    <li>python314Packages.fireworks-ai.dist</li>
    <li>python314Packages.formulaic</li>
    <li>python314Packages.formulaic.dist</li>
    <li>python314Packages.gradio</li>
    <li>python314Packages.gradio.dist</li>
    <li>python314Packages.great-tables</li>
    <li>python314Packages.great-tables.dist</li>
    <li>python314Packages.holistic-trace-analysis</li>
    <li>python314Packages.holistic-trace-analysis.dist</li>
    <li>python314Packages.holoviews</li>
    <li>python314Packages.holoviews.dist</li>
    <li>python314Packages.hvplot</li>
    <li>python314Packages.hvplot.dist</li>
    <li>python314Packages.igraph</li>
    <li>python314Packages.igraph.dist</li>
    <li>python314Packages.instructor</li>
    <li>python314Packages.instructor.dist</li>
    <li>python314Packages.intake</li>
    <li>python314Packages.intake.dist</li>
    <li>python314Packages.iplotx</li>
    <li>python314Packages.iplotx.dist</li>
    <li>python314Packages.itables</li>
    <li>python314Packages.itables.dist</li>
    <li>python314Packages.kmapper</li>
    <li>python314Packages.kmapper.dist</li>
    <li>python314Packages.langchain-azure-dynamic-sessions</li>
    <li>python314Packages.langchain-azure-dynamic-sessions.dist</li>
    <li>python314Packages.langchain-classic</li>
    <li>python314Packages.langchain-classic.dist</li>
    <li>python314Packages.langchain-community</li>
    <li>python314Packages.langchain-community.dist</li>
    <li>python314Packages.langchain-deepseek</li>
    <li>python314Packages.langchain-deepseek.dist</li>
    <li>python314Packages.langchain-experimental</li>
    <li>python314Packages.langchain-experimental.dist</li>
    <li>python314Packages.langchain-fireworks</li>
    <li>python314Packages.langchain-fireworks.dist</li>
    <li>python314Packages.langchain-mongodb</li>
    <li>python314Packages.langchain-mongodb.dist</li>
    <li>python314Packages.langchain-openai</li>
    <li>python314Packages.langchain-openai.dist</li>
    <li>python314Packages.langchain-perplexity</li>
    <li>python314Packages.langchain-perplexity.dist</li>
    <li>python314Packages.langchain-xai</li>
    <li>python314Packages.langchain-xai.dist</li>
    <li>python314Packages.langgraph-checkpoint-mongodb</li>
    <li>python314Packages.langgraph-checkpoint-mongodb.dist</li>
    <li>python314Packages.langgraph-checkpoint-sqlite</li>
    <li>python314Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python314Packages.langgraph-store-mongodb</li>
    <li>python314Packages.langgraph-store-mongodb.dist</li>
    <li>python314Packages.layoutparser</li>
    <li>python314Packages.layoutparser.dist</li>
    <li>python314Packages.leidenalg</li>
    <li>python314Packages.leidenalg.dist</li>
    <li>python314Packages.lifelines</li>
    <li>python314Packages.lifelines.dist</li>
    <li>python314Packages.linearmodels</li>
    <li>python314Packages.linearmodels.dist</li>
    <li>python314Packages.litellm</li>
    <li>python314Packages.litellm.dist</li>
    <li>python314Packages.llm</li>
    <li>python314Packages.llm-anthropic</li>
    <li>python314Packages.llm-anthropic.dist</li>
    <li>python314Packages.llm-cmd</li>
    <li>python314Packages.llm-cmd.dist</li>
    <li>python314Packages.llm-command-r</li>
    <li>python314Packages.llm-command-r.dist</li>
    <li>python314Packages.llm-deepseek</li>
    <li>python314Packages.llm-deepseek.dist</li>
    <li>python314Packages.llm-docs</li>
    <li>python314Packages.llm-docs.dist</li>
    <li>python314Packages.llm-echo</li>
    <li>python314Packages.llm-echo.dist</li>
    <li>python314Packages.llm-fragments-github</li>
    <li>python314Packages.llm-fragments-github.dist</li>
    <li>python314Packages.llm-fragments-pypi</li>
    <li>python314Packages.llm-fragments-pypi.dist</li>
    <li>python314Packages.llm-fragments-reader</li>
    <li>python314Packages.llm-fragments-reader.dist</li>
    <li>python314Packages.llm-fragments-symbex</li>
    <li>python314Packages.llm-fragments-symbex.dist</li>
    <li>python314Packages.llm-gemini</li>
    <li>python314Packages.llm-gemini.dist</li>
    <li>python314Packages.llm-gguf</li>
    <li>python314Packages.llm-gguf.dist</li>
    <li>python314Packages.llm-git</li>
    <li>python314Packages.llm-git.dist</li>
    <li>python314Packages.llm-github-copilot</li>
    <li>python314Packages.llm-github-copilot.dist</li>
    <li>python314Packages.llm-grok</li>
    <li>python314Packages.llm-grok.dist</li>
    <li>python314Packages.llm-groq</li>
    <li>python314Packages.llm-groq.dist</li>
    <li>python314Packages.llm-hacker-news</li>
    <li>python314Packages.llm-hacker-news.dist</li>
    <li>python314Packages.llm-jq</li>
    <li>python314Packages.llm-jq.dist</li>
    <li>python314Packages.llm-llama-server</li>
    <li>python314Packages.llm-llama-server.dist</li>
    <li>python314Packages.llm-mistral</li>
    <li>python314Packages.llm-mistral.dist</li>
    <li>python314Packages.llm-ollama</li>
    <li>python314Packages.llm-ollama.dist</li>
    <li>python314Packages.llm-openai-plugin</li>
    <li>python314Packages.llm-openai-plugin.dist</li>
    <li>python314Packages.llm-openrouter</li>
    <li>python314Packages.llm-openrouter.dist</li>
    <li>python314Packages.llm-pdf-to-images</li>
    <li>python314Packages.llm-pdf-to-images.dist</li>
    <li>python314Packages.llm-perplexity</li>
    <li>python314Packages.llm-perplexity.dist</li>
    <li>python314Packages.llm-sentence-transformers</li>
    <li>python314Packages.llm-sentence-transformers.dist</li>
    <li>python314Packages.llm-templates-fabric</li>
    <li>python314Packages.llm-templates-fabric.dist</li>
    <li>python314Packages.llm-templates-github</li>
    <li>python314Packages.llm-templates-github.dist</li>
    <li>python314Packages.llm-tools-datasette</li>
    <li>python314Packages.llm-tools-datasette.dist</li>
    <li>python314Packages.llm-tools-quickjs</li>
    <li>python314Packages.llm-tools-quickjs.dist</li>
    <li>python314Packages.llm-tools-simpleeval</li>
    <li>python314Packages.llm-tools-simpleeval.dist</li>
    <li>python314Packages.llm-tools-sqlite</li>
    <li>python314Packages.llm-tools-sqlite.dist</li>
    <li>python314Packages.llm-venice</li>
    <li>python314Packages.llm-venice.dist</li>
    <li>python314Packages.llm-video-frames</li>
    <li>python314Packages.llm-video-frames.dist</li>
    <li>python314Packages.llm.dist</li>
    <li>python314Packages.llmx</li>
    <li>python314Packages.llmx.dist</li>
    <li>python314Packages.marimo</li>
    <li>python314Packages.marimo.dist</li>
    <li>python314Packages.mistral-common</li>
    <li>python314Packages.mistral-common.dist</li>
    <li>python314Packages.msticpy</li>
    <li>python314Packages.msticpy.dist</li>
    <li>python314Packages.narwhals</li>
    <li>python314Packages.narwhals.dist</li>
    <li>python314Packages.nbsmoke</li>
    <li>python314Packages.nbsmoke.dist</li>
    <li>python314Packages.neurokit2</li>
    <li>python314Packages.neurokit2.dist</li>
    <li>python314Packages.niaarm</li>
    <li>python314Packages.niaarm.dist</li>
    <li>python314Packages.openai</li>
    <li>python314Packages.openai-agents</li>
    <li>python314Packages.openai-agents.dist</li>
    <li>python314Packages.openai.dist</li>
    <li>python314Packages.pandantic</li>
    <li>python314Packages.pandantic.dist</li>
    <li>python314Packages.pandas-stubs</li>
    <li>python314Packages.pandas-stubs.dist</li>
    <li>python314Packages.pandera</li>
    <li>python314Packages.pandera.dist</li>
    <li>python314Packages.panel</li>
    <li>python314Packages.panel.dist</li>
    <li>python314Packages.pdfplumber</li>
    <li>python314Packages.pdfplumber.dist</li>
    <li>python314Packages.plotly</li>
    <li>python314Packages.plotly.dist</li>
    <li>python314Packages.posthog</li>
    <li>python314Packages.posthog.dist</li>
    <li>python314Packages.pyreadstat</li>
    <li>python314Packages.pyreadstat.dist</li>
    <li>python314Packages.sasmodels</li>
    <li>python314Packages.sasmodels.dist</li>
    <li>python314Packages.shiny</li>
    <li>python314Packages.shiny.dist</li>
    <li>python314Packages.skops</li>
    <li>python314Packages.skops.dist</li>
    <li>python314Packages.smolagents</li>
    <li>python314Packages.smolagents.dist</li>
    <li>python314Packages.sqlframe</li>
    <li>python314Packages.sqlframe.dist</li>
    <li>python314Packages.sqlite-vec</li>
    <li>python314Packages.sqlite-vec.dist</li>
    <li>python314Packages.st-pages</li>
    <li>python314Packages.st-pages.dist</li>
    <li>python314Packages.streamlit</li>
    <li>python314Packages.streamlit-avatar</li>
    <li>python314Packages.streamlit-avatar.dist</li>
    <li>python314Packages.streamlit-card</li>
    <li>python314Packages.streamlit-card.dist</li>
    <li>python314Packages.streamlit-echarts</li>
    <li>python314Packages.streamlit-echarts.dist</li>
    <li>python314Packages.streamlit-folium</li>
    <li>python314Packages.streamlit-folium.dist</li>
    <li>python314Packages.streamlit-kpi-card</li>
    <li>python314Packages.streamlit-kpi-card.dist</li>
    <li>python314Packages.streamlit-notify</li>
    <li>python314Packages.streamlit-notify.dist</li>
    <li>python314Packages.streamlit.dist</li>
    <li>python314Packages.synergy</li>
    <li>python314Packages.synergy.dist</li>
    <li>python314Packages.unstructured-inference</li>
    <li>python314Packages.unstructured-inference.dist</li>
    <li>python314Packages.vega</li>
    <li>python314Packages.vega.dist</li>
    <li>quark-engine</li>
    <li>quark-engine.dist</li>
    <li>rembg</li>
    <li>rembg.dist</li>
    <li>seagoat</li>
    <li>seagoat.dist</li>
    <li>shell-genie</li>
    <li>shell-genie.dist</li>
    <li>shell-gpt</li>
    <li>shell-gpt.dist</li>
    <li>sploitscan</li>
    <li>sploitscan.dist</li>
    <li>tandoor-recipes</li>
    <li>textlsp</li>
    <li>textlsp.dist</li>
    <li>tftui</li>
    <li>tftui.dist</li>
    <li>theharvester</li>
    <li>theharvester.dist</li>
    <li>tocpdf</li>
    <li>tocpdf.dist</li>
    <li>tts</li>
    <li>tts.dist</li>
    <li>unstructured-api</li>
    <li>vectorcode</li>
    <li>vectorcode.dist</li>
    <li>vimPlugins.vectorcode-nvim</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>lasuite-docs</summary>
<pre>    return _bootstrap._gcd_import(name[level:], package, level)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1387, in _gcd_import
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1360, in _find_and_load
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1331, in _find_and_load_unlocked
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 919, in _load_unlocked
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 879, in _load_backward_compatible
  File &quot;/nix/store/nnjxc498g31s7gsygp7ypmhx9par0hnk-python3.13-django-configurations-2.5.1/lib/python3.13/site-packages/configurations/importer.py&quot;, line 152, in load_module
    self.spec.loader.exec_module(mod)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File &quot;&lt;frozen importlib._bootstrap_external&gt;&quot;, line 1023, in exec_module
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 488, in _call_with_frames_removed
  File &quot;/nix/var/nix/builds/nix-86480-1459905759/source/src/backend/impress/settings.py&quot;, line 956, in &lt;module&gt;
    class Production(Base):
    ...&lt;74 lines&gt;...
        }
  File &quot;/nix/var/nix/builds/nix-86480-1459905759/source/src/backend/impress/settings.py&quot;, line 971, in Production
    gethostbyname(gethostname()),
    ~~~~~~~~~~~~~^^^^^^^^^^^^^^^
socket.gaierror: [Errno 8] nodename nor servname provided, or not known</pre>
</details>
<details>
<summary>python313Packages.ax-platform</summary>
<pre>  invalid value encountered in sqrt

ax/utils/testing/tests/test_mock.py::TestMock::test_optimize_with_nsgaii_mocks
  /nix/store/iq6f4fr1al2jj764r5mqfi1hm9nni7ix-python3.13-botorch-0.16.1/lib/python3.13/site-packages/botorch/acquisition/analytic.py:340: NumericsWarning:
  
  ExpectedImprovement has known numerical issues that lead to suboptimal optimization performance. It is strongly recommended to simply replace
  
  	 ExpectedImprovement 	 --&gt; 	 LogExpectedImprovement 
  
  instead, which fixes the issues and has the same API. See https://arxiv.org/abs/2310.20708 for details.

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED ax/analysis/plotly/tests/test_p_feasible.py::TestPFeasiblePlot::test_offline - TimeoutError: Test took longer than 120 seconds. To see a profiler output, set `TestCase.PROFILE_TESTS` to `True`. To specify a reason for a long running test, utilize the @ax_long_test decorator. If your test is long because it&#x27;s doing modeling, please use the @mock_botorch_optimize decorator and see if that helps.
FAILED ax/analysis/plotly/tests/test_p_feasible.py::TestPFeasiblePlot::test_online - TimeoutError: Test took longer than 120 seconds. To see a profiler output, set `TestCase.PROFILE_TESTS` to `True`. To specify a reason for a long running test, utilize the @ax_long_test decorator. If your test is long because it&#x27;s doing modeling, please use the @mock_botorch_optimize decorator and see if that helps.
FAILED ax/analysis/tests/test_overview.py::TestOverview::test_offline - AssertionError: &lt;ax.analysis.analysis_card.ErrorAnalysisCard object at 0x1302c9550&gt; is an instance of &lt;class &#x27;ax.analysis.analysis_card.ErrorAnalysisCard&#x27;&gt;
FAILED ax/analysis/tests/test_overview.py::TestOverview::test_online - AssertionError: 1 != 0
FAILED ax/analysis/tests/test_results.py::TestResultsAnalysis::test_offline_experiments - TimeoutError: Test took longer than 120 seconds. To see a profiler output, set `TestCase.PROFILE_TESTS` to `True`. To specify a reason for a long running test, utilize the @ax_long_test decorator. If your test is long because it&#x27;s doing modeling, please use the @mock_botorch_optimize decorator and see if that helps.
FAILED ax/generation_strategy/tests/test_generation_strategy.py::TestGenerationStrategy::test_deduplication_and_fallback - TimeoutError: Test took longer than 120 seconds. To see a profiler output, set `TestCase.PROFILE_TESTS` to `True`. To specify a reason for a long running test, utilize the @ax_long_test decorator. If your test is long because it&#x27;s doing modeling, please use the @mock_botorch_optimize decorator and see if that helps.
== 6 failed, 1513 passed, 111 deselected, 3533 warnings in 2370.54s (0:39:30) ==</pre>
</details>
<details>
<summary>python314Packages.banks</summary>
<pre>        read_value = os.environ.get(f&quot;{prefix}{name}&quot;)
        if read_value is None:
            return original_value
    
        # Convert string from env var to the actual type
&gt;       t = super().__getattribute__(&quot;__annotations__&quot;)[name]
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
E       AttributeError: &#x27;_BanksConfig&#x27; object has no attribute &#x27;__annotations__&#x27;. Did you mean: &#x27;__annotate_func__&#x27;?

/nix/store/0ph53iynnf09n72z7mihfxxjy2lmp0iy-python3.14-banks-2.2.0/lib/python3.14/site-packages/banks/config.py:31: AttributeError
=============================== warnings summary ===============================
../../../../../store/nxafi194px2ln6kl8rxfgj918z9v031p-python3.14-litellm-1.80.0/lib/python3.14/site-packages/litellm/litellm_core_utils/logging_utils.py:273: 17 warnings
  /nix/store/nxafi194px2ln6kl8rxfgj918z9v031p-python3.14-litellm-1.80.0/lib/python3.14/site-packages/litellm/litellm_core_utils/logging_utils.py:273: DeprecationWarning: &#x27;asyncio.iscoroutinefunction&#x27; is deprecated and slated for removal in Python 3.16; use inspect.iscoroutinefunction() instead
    if asyncio.iscoroutinefunction(func):

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/test_config.py::test_config_env_override - AttributeError: &#x27;_BanksConfig&#x27; object has no attribute &#x27;__annotations__&#x27;. D...
FAILED tests/test_config.py::test_config_env_prefix - AttributeError: &#x27;_BanksConfig&#x27; object has no attribute &#x27;__annotations__&#x27;. D...
====== 2 failed, 93 passed, 14 skipped, 1 xfailed, 17 warnings in 16.31s =======</pre>
</details>
<details>
<summary>python314Packages.livelossplot</summary>
<pre>    controller.open(&quot;file://&quot; + filename, new=NEW_PARAM[new])
/nix/store/0iw3akzx6ygiqzpxi9s422sc2fl6190g-python3-3.14.2/lib/python3.14/webbrowser.py:93: in open
    if browser.open(url, new, autoraise):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
/nix/store/0iw3akzx6ygiqzpxi9s422sc2fl6190g-python3-3.14.2/lib/python3.14/webbrowser.py:652: in open
    rc = osapipe.close()
         ^^^^^^^^^^^^^^^
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = &lt;os._wrap_close object at 0x10dae5e80&gt;

&gt;   ???
E   BrokenPipeError: [Errno 32] Broken pipe

&lt;frozen os&gt;:1050: BrokenPipeError
----------------------------- Captured stderr call -----------------------------
/nix/store/yi3x6xpilh0bq0bgb61imy6gpv8b1la4-bash-5.3p3/bin/sh: line 1: osascript: command not found
=========================== short test summary info ============================
FAILED tests/test_bokeh_output.py::test_bokeh_plot - BrokenPipeError: [Errno 32] Broken pipe
======================== 1 failed, 12 passed in 32.39s =========================</pre>
</details>
<details>
<summary>python314Packages.ploomber-core</summary>
<pre>tests/telemetry/test_telemetry.py:823
  /nix/var/nix/builds/nix-86480-1459905763/source/tests/telemetry/test_telemetry.py:823: PytestUnknownMarkWarning: Unknown pytest.mark.allow_posthog - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html
    @pytest.mark.allow_posthog

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/test_config.py::test_stores_defaults - AttributeError: &#x27;MyConfig&#x27; object has no attribute &#x27;__annotations__&#x27;. Did y...
FAILED tests/test_config.py::test_reads_existing - AttributeError: &#x27;MyConfig&#x27; object has no attribute &#x27;__annotations__&#x27;. Did y...
FAILED tests/test_config.py::test_ignores_extra - AttributeError: &#x27;MyConfig&#x27; object has no attribute &#x27;__annotations__&#x27;. Did y...
FAILED tests/test_config.py::test_stores_on_update - AttributeError: &#x27;MyConfig&#x27; object has no attribute &#x27;__annotations__&#x27;. Did y...
FAILED tests/test_config.py::test_uses_default_if_missing - AttributeError: &#x27;MyConfig&#x27; object has no attribute &#x27;__annotations__&#x27;. Did y...
FAILED tests/test_config.py::test_uses_defaults_if_corrupted[not yaml] - Failed: DID NOT WARN. No warnings of type (&lt;class &#x27;UserWarning&#x27;&gt;,) were emi...
FAILED tests/test_config.py::test_uses_defaults_if_corrupted[[[]] - Failed: DID NOT WARN. No warnings of type (&lt;class &#x27;UserWarning&#x27;&gt;,) were emi...
FAILED tests/test_config.py::test_config_with_factory - AttributeError: &#x27;AnotherConfig&#x27; object has no attribute &#x27;__annotations__&#x27;. ...
FAILED tests/test_config.py::test_factory_keeps_existing_values - AttributeError: &#x27;AnotherConfig&#x27; object has no attribute &#x27;__annotations__&#x27;. ...
FAILED tests/test_config.py::test_restores_values_if_wrong_type[content0-42-value] - Failed: DID NOT WARN. No warnings of type (&lt;class &#x27;UserWarning&#x27;&gt;,) were emi...
FAILED tests/test_config.py::test_restores_values_if_wrong_type[content1-42-value] - Failed: DID NOT WARN. No warnings of type (&lt;class &#x27;UserWarning&#x27;&gt;,) were emi...
FAILED tests/test_config.py::test_race_condition - AttributeError: &#x27;MyConfig&#x27; object has no attribute &#x27;__annotations__&#x27;
FAILED tests/test_config.py::test_readonly - AttributeError: &#x27;MyConfig&#x27; object has no attribute &#x27;__annotations__&#x27;. Did y...
========== 13 failed, 90 passed, 118 deselected, 2 warnings in 3.46s ===========</pre>
</details>
<details>
<summary>python314Packages.reflex</summary>
<pre>  /nix/var/nix/builds/nix-86480-1459905820/source/reflex/utils/types.py:382: PydanticDeprecatedSince20:
  
  The `__fields__` attribute is deprecated, use the `model_fields` class property instead. Deprecated in Pydantic V2.0 to be removed in V3.0. See Pydantic V2 Migration Guide at https://errors.pydantic.dev/2.12/migration/

tests/units/test_attribute_access_type.py: 10 warnings
  /nix/var/nix/builds/nix-86480-1459905820/source/reflex/utils/types.py:384: PydanticDeprecatedSince20:
  
  The `__fields__` attribute is deprecated, use the `model_fields` class property instead. Deprecated in Pydantic V2.0 to be removed in V3.0. See Pydantic V2 Migration Guide at https://errors.pydantic.dev/2.12/migration/

tests/units/test_model.py::test_upcast_event_handler_arg[handler0-payload0]
  /nix/var/nix/builds/nix-86480-1459905820/source/reflex/state.py:1910: PydanticDeprecatedSince20:
  
  The `__fields__` attribute is deprecated, use the `model_fields` class property instead. Deprecated in Pydantic V2.0 to be removed in V3.0. See Pydantic V2 Migration Guide at https://errors.pydantic.dev/2.12/migration/

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/units/test_state.py::test_state_manager_lock_expire - reflex.utils.exceptions.LockExpiredError: Lock expired for token 445ad475-7...
FAILED tests/units/test_state.py::test_state_manager_lock_expire_contend - reflex.utils.exceptions.LockExpiredError: Lock expired for token 301f46ba-0...
FAILED tests/units/test_state.py::test_state_manager_lock_warning_threshold_contend - reflex.utils.exceptions.LockExpiredError: Lock expired for token 19561524-8...
= 3 failed, 3501 passed, 17 skipped, 31 deselected, 76 warnings in 68.00s (0:01:08) =</pre>
</details>
<details>
<summary>python314Packages.wandb</summary>
<pre>Setting COVERAGE_FILE to /nix/var/nix/builds/nix-86480-1459905823/source/.coverage
=============================== warnings summary ===============================
../../../../../store/2p20k2cy8cv4bnh90lj5lma3fzwf17r5-python3.14-pydantic-2.12.4/lib/python3.14/site-packages/pydantic/_internal/_generate_schema.py:2249: 21 warnings
  /nix/store/2p20k2cy8cv4bnh90lj5lma3fzwf17r5-python3.14-pydantic-2.12.4/lib/python3.14/site-packages/pydantic/_internal/_generate_schema.py:2249: UnsupportedFieldAttributeWarning: The &#x27;repr&#x27; attribute with value False was provided to the `Field()` function, which has no effect in the context it was used. &#x27;repr&#x27; is field-specific metadata, and can only be attached to a model field using `Annotated` metadata or by assignment. This may have happened because an `Annotated` type alias using the `type` statement was used, or if the `Field()` function was attached to a single member of a union type.
    warnings.warn(

../../../../../store/2p20k2cy8cv4bnh90lj5lma3fzwf17r5-python3.14-pydantic-2.12.4/lib/python3.14/site-packages/pydantic/_internal/_generate_schema.py:2249: 21 warnings
  /nix/store/2p20k2cy8cv4bnh90lj5lma3fzwf17r5-python3.14-pydantic-2.12.4/lib/python3.14/site-packages/pydantic/_internal/_generate_schema.py:2249: UnsupportedFieldAttributeWarning: The &#x27;frozen&#x27; attribute with value True was provided to the `Field()` function, which has no effect in the context it was used. &#x27;frozen&#x27; is field-specific metadata, and can only be attached to a model field using `Annotated` metadata or by assignment. This may have happened because an `Annotated` type alias using the `type` statement was used, or if the `Field()` function was attached to a single member of a union type.
    warnings.warn(

tests/unit_tests/test_wandb_settings_old.py::test__global_path_env_var_exists_and_is_writable
  /nix/var/nix/builds/nix-86480-1459905823/source/tests/unit_tests/test_wandb_settings_old.py:33: PytestWarning: Value of environment variable WANDB_CONFIG_DIR type should be str, but got PosixPath(&#x27;/nix/var/nix/builds/nix-86480-1459905823/pytest-of-_nixbld10/pytest-0/popen-gw5/test__global_path_env_var_exis0/config&#x27;) (type: PosixPath); converted to str implicitly
    monkeypatch.setenv(&quot;WANDB_CONFIG_DIR&quot;, config_dir)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/unit_tests/test_torch.py::test_watch_parameters_torch_jit[parameters] - AttributeError: &#x27;Linear&#x27; object has no attribute &#x27;__annotations__&#x27;. Did you mean: &#x27;__annotate_func__&#x27;?
FAILED tests/unit_tests/test_torch.py::test_watch_parameters_torch_jit[all] - AttributeError: &#x27;Linear&#x27; object has no attribute &#x27;__annotations__&#x27;. Did you mean: &#x27;__annotate_func__&#x27;?
FAILED tests/unit_tests/test_torch.py::test_watch_graph_torch_jit - AttributeError: &#x27;Net&#x27; object has no attribute &#x27;__annotations__&#x27;. Did you mean: &#x27;__annotate_func__&#x27;?
= 3 failed, 1801 passed, 142 skipped, 3 xfailed, 5 xpassed, 43 warnings in 168.41s (0:02:48) =</pre>
</details>
</details>

--- a/pkgs/development/python-modules/openai/default.nix
+++ b/pkgs/development/python-modules/openai/default.nix
@@ -51,14 +51,14 @@
 
 buildPythonPackage rec {
   pname = "openai";
-  version = "2.11.0";
+  version = "2.15.0";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "openai";
     repo = "openai-python";
     tag = "v${version}";
-    hash = "sha256-f4d6gz4W04NPu43u0ovORzvF8ZW7oIYO8DJIqUGIEdE=";
+    hash = "sha256-FmOmWtkp+Pe3pICi0uvzsb1nsP4M4yc/34U3Bd2q/KE=";
   };
 
   postPatch = ''substituteInPlace pyproject.toml --replace-fail "hatchling==1.26.3" "hatchling"'';

